stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -40,6 +40,25 @@ Base.compilecache(Base.identify_package("TestPkg"))
     popfirst!(Base.LOAD_PATH)
 end
 
+using Test, Random, LinearAlgebra
+
+# Test that modules are loaded on workers despite a using; addprocs; using chain
+# see https://github.com/JuliaLang/julia/issues/28859
+# TODO: We probably want a non stdlib package in these tests
+for w in workers()
+    mods = remotecall_fetch(()->map(string, values(Base.loaded_modules)), w)
+    @test "Test" in mods
+    @test "Random" in mods
+    @test "LinearAlgebra" in mods
+end
+
+# Make sure that call of imported binding (on master)
+# works everywhere.
+for w in procs()
+    remotecall_wait(seed, 1)
```

Comment:
`seed` isn't a thing. Did you mean `Random.seed!`?

Reply:
Yes :/